packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -295,7 +295,9 @@ class Tooltip extends StatefulWidget {
   }
 }
 
-class _TooltipState extends State<Tooltip> with SingleTickerProviderStateMixin {
+/// Contains the state for a [Tooltip]. This class can be used to
```

Comment:
Our style guide suggests:
`The first paragraph of any dartdoc section must be a short self-contained sentence`

Reply:
Thanks for the review! Made the changes.